coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -15,15 +15,31 @@
         v-for="(item, index) in orderBy(taskItems, 'order')"
         v-bind:key="index"
         expand-icon=""
-        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item) }">
+        :class="{ 'disabled': !item.enabled, 'draft': isDraft(item),
+        'awaitConfirm': !confirmCheckbox && isConfirmEnabled(item.type, item.status)}">
 
         <v-expansion-panel-header class="no-dropdown">
           <div class="list-item">
             <div class="filing-type">
               <div class="list-item__title">{{item.title}}</div>
+              <div class="bcorps-ar-subtitle"
+                 v-if="entityFilter(EntityTypes.BCORP) &&
+                 isConfirmEnabled(item.type, item.status)"
+              >
+                <p>Verify your Office Address and Current Directors before filing your Annual Report.</p>
+                <v-checkbox
+                  id="confirm-checkbox"
+                  class="todo-list-checkbox"
+                  label="All information about the Office Addresses and Current Directors is correct."
+                  :disabled=!item.enabled
+                  v-model=confirmCheckbox
+                  @click.native.stop
+                />
+              </div>
               <div class="list-item__subtitle">
-                <div v-if="item.subtitle" class="todo-status">{{item.subtitle}}</div>
-
+                <div v-if="item.subtitle && !entityFilter(EntityTypes.BCORP)" class="todo-status">
```

Comment:
For now I'd prefer to see this element _included_ for Coops only instead of _excluded_ for BCorps.
The way I think of this is that I know what I know (ie, it's needed for Coops) but I don't know that it applies to everything except BCorps. Today the latter might be true but it will be an assumption for any future entity type, instead of considering future entity types separately.

Reply:
Ok, that makes perfect sense to me. I will change this!